tests-ci/generate.sh

```diff
@@ -161,7 +161,7 @@ elif [[ $application == 'fullapp' ]]; then
   "spring-boot-devtools" \
   "logstash" \
   "jib" \
-  "dockerfile" \
+  "dockerfile-maven" \
```

Comment:
Don't forget to also add dockerfile-gradle for the gradleapp :-)

And if https://github.com/jhipster/jhipster-lite/pull/8986 is merged before this PR:
```suggestion
  "dockerfile-${java_build_tool}" \
```

Reply:
> Don't forget to also add dockerfile-gradle for the gradleapp :-)
And if https://github.com/jhipster/jhipster-lite/pull/8986 is merged before this PR

Thanks, it's done. I remembered the merge request ;)